### PR TITLE
Add support to include multiple toolchain aliases

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci build toolchain macOS

**Comment by shahmishal** [comment]
Started PR testing with `darwin_toolchain_alias=PR63924,swift-PR-63924-601`

**Comment by shahmishal** [comment]
@swift-ci test

**Comment by AnthonyLatsis** [comment]
Thanks! I can tell that it most likely works but I’m not able to convince the system to allow me to run something from the toolchain.

File: utils/build-script-impl [hunk 1]
```diff
@@ -3505,7 +3505,15 @@ function build_and_test_installable_package() {
           call ${PLISTBUDDY_BIN} -c "Add CFBundleIdentifier string '${DARWIN_TOOLCHAIN_BUNDLE_IDENTIFIER}'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
           call ${PLISTBUDDY_BIN} -c "Add ReportProblemURL string '${DARWIN_TOOLCHAIN_REPORT_URL}'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
           call ${PLISTBUDDY_BIN} -c "Add Aliases array" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
-          call ${PLISTBUDDY_BIN} -c "Add Aliases:0 string '${DARWIN_TOOLCHAIN_ALIAS}'" "${DARWIN_TOOLCHAIN_INFO_PLIST}"
+
+          aliases_count=0
+          IFS=","
```
[the remark this answers is not in the document]
Comment (shahmishal) [reply utils/build-script-impl]: Good point, updated. 
https://github.com/apple/swift/pull/63924/commits/4d3eebe529684d47fdc565c77834437f5856957c

**Comment by shahmishal** [comment]
@swift-ci build toolchain macOS

**Pull request merged**